### PR TITLE
AmateureExtreme xPath scene scraper

### PR DESCRIPTION
**Pull request by DogmaDragon**
Adding `01-01` to the date since the website only provides the year. 

Requested here https://github.com/stashapp/CommunityScrapers/issues/73#issuecomment-1186344468

### CODE REVIEW

File: scrapers/AmateureExtreme.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      Title: //h1[@class='info-section--title']
+      URL: //link[@rel="canonical"]/@href
+      Date:
+        selector: //div[@class='posted-info meta-description small']/div[@class='float-left']/span[@class='text-nowrap']/span[@class='fa5-text']
+        postProcess:
+          - replace:
+              - regex: $
+                with: $1-01-01
+      Tags:
+        Name: //div[@class='tags mt-3']/a | //div[@class='posted-info meta-description small']/div[@class='float-left']/span[2]/span[@class='fa5-text']/a
```
Comment (bnkai) [inline scrapers/AmateureExtreme.yml:19]: simpler ?
```yaml
        Name: //a[contains(@href,'collections?')]
```

File: scrapers/AmateureExtreme.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  sceneScraper:
+    scene:
+      Title: //h1[@class='info-section--title']
+      URL: //link[@rel="canonical"]/@href
+      Date:
+        selector: //div[@class='posted-info meta-description small']/div[@class='float-left']/span[@class='text-nowrap']/span[@class='fa5-text']
+        postProcess:
+          - replace:
+              - regex: $
+                with: $1-01-01
```
Comment (bnkai) [inline scrapers/AmateureExtreme.yml:17]:
```suggestion
                with: -01-01
```
No match group is used so `$1` doesnt have a value anyway

Comment (DogmaDragon) [reply scrapers/AmateureExtreme.yml:17]: Thank you.

File: scrapers/AmateureExtreme.yml (hunk 1 above)
Comment (DogmaDragon) [reply scrapers/AmateureExtreme.yml:19]: Tested, works. I need to get more familiar with contains...

**Pull request merged**